SitumWayfinding/Classes/WayfindingLibrary/WayfindingError.swift

```diff
@@ -43,6 +49,8 @@ extension WayfindingError {
             return NSLocalizedString("wayfindingError.buildingNotExist", bundle: SitumMapsLibrary.bundle, comment: "")
         case .requestError:
             return NSLocalizedString("wayfindingError.requestError", bundle: SitumMapsLibrary.bundle, comment: "")
+        case .offlineTilesNotDownloaded:
+            return NSLocalizedString("wayfindingError.offlineTilesNotDownloaded", bundle: SitumMapsLibrary.bundle, comment: "")
```

Comment:
Ten sentido localizar estos errores?

Reply:
Creo que facilita a vida se tes que tratar con NSErrors proporcionados polo sdk e o Error de Swift que xera WYF

Reply:
A conversión de erros si que a vexo ben, pero refierome a que esos son máis erros de programación que erros que un usuario final vai a ver, por eso preguntaba